### PR TITLE
Add a unit test for testing validation of token which are signed by HS family algorithms 

### PR DESCRIPTION
**Pull request by adibrastegarnia**

### CODE REVIEW

File: pkg/auth/jwt_test.go [hunk 1]
```diff
@@ -93,8 +93,11 @@ const dexkeys = `{
...
+const sampleTokenHS256Signature = `eyJhbGciOiJIUzI1NiIsImtpZCI6ImRmMzYyZGMzNzRjNWJmMWEyZjEwMzllZDhkNjYyOWM5MGI5M2EwNGUifQ.eyJpc3MiOiJodHRwOi8vZGV4OjMyMDAwIiwic3ViIjoiQ2lRd09HRTROamcwWWkxa1lqZzRMVFJpTnpNdE9UQmhPUzB6WTJReE5qWXhaalUwTmpnU0JXeHZZMkZzIiwiYXVkIjoib25vcy1ndWkiLCJleHAiOjE5MDk5Mzg0NDcsImlhdCI6MTU5NDU3ODQ0Nywibm9uY2UiOiJjelJ2TmtoTFVYbFhVSE5EYWpsMmNHSlFjMVZOUld0T2NERmtMbkJ0ZHkxbFJTMVlkM0YzVUVGNE5IZFEiLCJhdF9oYXNoIjoiWXcxQ1M1UmdvYVRVWUNUZWtxUkxBZyIsImVtYWlsIjoic2VhbkBvcGVubmV0d29ya2luZy5vcmciLCJlbWFpbF92ZXJpZmllZCI6dHJ1ZSwibmFtZSI6InNlYW4ifQ.vAx1WCxsJNEvpfMvTU0IqnWgen1xmS3lOjkorZL5uWU`
```
Comment (SeanCondon) [inline pkg/auth/jwt_test.go:100]: I don't  see why a second one is required

File: pkg/auth/jwt.go [hunk 2]
```diff
@@ -43,6 +44,9 @@ const (
 	OIDCServerURL = "OIDC_SERVER_URL"
 	// OpenidConfiguration is the discovery point on the OIDC server
 	OpenidConfiguration = ".well-known/openid-configuration"
+
+	// RSAPublicKey an RSA public key
+	RSAPublicKey = "RSA_PUBLIC_KEY"
```
Comment (SeanCondon) [inline pkg/auth/jwt.go:49]: I don't see what this is for

Comment (adibrastegarnia) [reply pkg/auth/jwt.go:49]: I should remove it. I don't know why it returned back in my branch.

File: pkg/auth/jwt_test.go (hunk 1 above)
Comment (adibrastegarnia) [reply pkg/auth/jwt_test.go:100]: The signature is different because the algorithm which is used is HS256

Comment (adibrastegarnia) [reply pkg/auth/jwt_test.go:100]: For testing of this part of the code: https://github.com/onosproject/onos-lib-go/blob/master/pkg/auth/jwt.go#L62

**Comment by adibrastegarnia** [comment]
@SeanCondon 
Thanks for your review.

**Comment by adibrastegarnia** [comment]
@SeanCondon 
Let me check something in my commits.

**Pull request merged**